frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/sc-node-list.tsx

```diff
@@ -107,10 +159,10 @@ const AttachedDevicesNodeTable: React.FC<NodeTableProps> = (props) => {
         {...props}
```

Comment:
```suggestion
data={data}
loaded={loaded}
```

Reply:
It's the same thing.

Reply:
The `props` does not provide much information. Since not many props are there, we can directly pass whatever props we have.